README.md

```diff
@@ -137,6 +135,22 @@ For example: `$BASE16_THEME_DEFAULT="solarized-light"`
 Open a new shell and type `base16` followed by a tab to perform tab
 completion.
 
+### Base16-shell hooks
+
+You can create your own base16-shell hooks. These scripts will execute
+every time you use base16-shell to change your theme. When a theme is
+changed via the command line aliases prefixes `base16_` (Bash/ZSH) and
+`base16-` (Fish) all executable `*.sh` (Bash/ZSH) or `*.fish` (Fish)
+files will be sourced. When combining this with the `$BASE16_THEME`
```

Comment:
I can't read this, something seems off.  Seems like osmething is missing int he middle.